addons/binding/org.openhab.binding.systeminfo.test/src/test/groovy/org/openhab/binding/systeminfo/test/SysteminfoOSGiTest.groovy

```diff
@@ -265,7 +316,15 @@ class SysteminfoOSGiTest extends OSGiTest{
         String channnelID = SysteminfoBindingConstants.CHANNEL_CPU_LOAD_1
         String acceptedItemType = "Number";
 
+        DecimalType mockCpuLoad1Value = new DecimalType(1.1)
+        when(mockedSystemInfo.getCpuLoad1()).thenReturn(mockCpuLoad1Value);
+
         initializeThingWithChannel(channnelID,acceptedItemType);
+
+        ChannelUID cpuLoad1ChannelUID = systemInfoThing.getChannel(SysteminfoBindingConstants.CHANNEL_CPU_LOAD_1).getUID()
+        SysteminfoHandler thingHandler = getThingHandler(SysteminfoHandler.class)
+
+        assertEquals(mockCpuLoad1Value, thingHandler.getInfoForChannel(cpuLoad1ChannelUID))
         assertItemState(acceptedItemType,DEFAULT_TEST_ITEM_NAME,DEFAULT_CHANNEL_TEST_PRIORITY);
```

Comment:
You can simply pass the `mockCpuLoad1Value `. Take a look at the `assertItemState` method. Now it checks that the item state is not NULL, you can simply change this assertion.